src/deck-options.md

```diff
@@ -2,262 +2,223 @@
 
 <!-- toc -->
 
-Deck options primarily control the way Anki schedules cards. It is recommended
+Deck options primarily control the way Anki schedules cards. If you're a new user, it's recommended
```

Comment:
```suggestion
Deck options primarily control the way Anki schedules cards. It is recommended
```
I'm recommending this stay the same and here's why -- tons of folks think they aren't new users, but if they are seeing this page or considering changing options for the first time, it's probably still too soon for them to be making changes!